QualityControl/public/layout/Layout.js

```diff
@@ -313,8 +313,8 @@ export default class Layout extends Observable {
     const { isOfficial } = this.model.services.layout.list.payload.find((item) => item.id === id);
 
     await this.model.services.layout.patchLayout(id, { isOfficial: !isOfficial });
-    await this.model.services.layout.getLayouts(this);
-    await this.model.services.layout.getLayoutsByUserId(this.model.session.personid, this);
+    await this.model.services.layout.getLayoutCards(this);
+    await this.model.services.layout.getLayoutCardsByUserId(this.model.session.personid, this);
```

Comment:
I believe here we can improve and drop the second promise. (I know it was like this already, but I see no reason not to improve it)
Basically the first promise request will return you all layouts. Thus, you can create your list of layoutsByUserId from that promise.